include/sidekiq/sidekiq_tx.h

```diff
@@ -37,14 +37,17 @@ namespace gr {
 		public:
 			typedef std::shared_ptr<sidekiq_tx> sptr;
 			static sptr make(
-					double sample_rate,
-					double attenuation,
-					double frequency,
-					double bandwidth,
-					int sync_type,
-					bool suppress_tune_transients,
-					uint8_t dataflow_mode,
-					int buffer_size);
+                int input_card_number,
+                int port1_handle,
+                int port2_handle,
```

Comment:
is this always going to be `100` or the `None` identifier? If so can it be removed from here?

Reply:
Ok, good catch again, I changed tx to only have one handle.